proposal/README.md

```diff
@@ -1 +1,94 @@
-proposal/requirement
+SUSE SolidDriver Checks
+=======================
+
+The purpose of the SUSE SolidDriver Checks is to ensure best end user
+experience by verifying that third party delivered drivers are built and
+deployed in a SUSE SolidDriver compliant] manner.
+
+General Features
+----------------
+*NOTE: Not all features are implemented at this time*
+
+* This tool shall be usable by SUSE, SUSE Partners, and SUSE customers.
+* The tool shall be able check running and installed modules on local or
+  remote systems
+* The tool shall be able check kernel module packages (KMPs) that have not
+  been installed
+* The tool shall create reports in the following formats:
+    - HTML
+    - Excel spreadsheet
+    - PDF
+    - JSON
+    - Terminal
+* Remote checks executed via ssh preferably using ssh keys for
+  authentication
+
+
+Checks
+------
+*NOTE: Not all checks are implemented at this time*
+
+* Critical checks failures shall give a high level warning (RED)
+* Important checks failures shall give a low level warning (AMBER)
```

Comment:
To align with SUSE brand color, maybe we should use ![image](https://user-images.githubusercontent.com/11676923/122156043-b50df700-ce57-11eb-8844-5c5d378cfcb2.png) Persimmon #FE7C3F for important checks failures?

Reply:
Actually we should probably use the color pallet from https://suse.eosdesignsystem.com/colors which is more tailored to software than the core corporate branding pallet. Yellow for warning and Red for danger are defined there.